static/app/utils/environment.tsx

```diff
@@ -3,13 +3,18 @@ import type {Environment} from 'sentry/types/project';
 const DEFAULT_EMPTY_ROUTING_NAME = 'none';
 const DEFAULT_EMPTY_ENV_NAME = '(No Environment)';
 
+/**
+ * Double encodes the environment name or display name to be used in the URL routing name.
+ * This is necessary because the environment name or display name may contain special characters
+ * that need to be encoded for the URL.
+ */
 export function getUrlRoutingName(env: Partial<Environment>) {
```

Comment:
Oh If we’re already encoding I would have just assumed this would have worked 🤔 did this actually fix it so we could delete that example environment ?

Reply:
Yeah I tested locally with a bunch of cases (with %, with /, without any special characters) and double encoding let it go through. 

So I thought maybe then we could just keep single encoding and remove the decoding within the endpoint since there seemed to be some middle layer decoding before then, but then when I did that it worked for the names with %, but failed for names with slashes in them (the case the decoding was originally added for) 🤷🏽‍♀️